client/src/components/Charts/DayCounter.jsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from 'react';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import styled from 'styled-components';
+
+dayjs.extend(relativeTime);
+
+const URL = process.env.REACT_APP_URL;
+const USER_ID = 9999;
+
+const DayCounter = () => {
+  const [lastSessionDate, setLastSessionDate] = useState();
+  const [dateDifference, setDateDifference] = useState();
+
+  const calculateDateDifference = () => {
+    const difference = dayjs(lastSessionDate).fromNow(true);
+    setDateDifference(difference);
+  };
+
+  const fetchDayOfLastBoulderSession = () => {
+    fetch(`${URL}/api/climbed_boulders_days/${USER_ID}/`)
+      .then(res => res.json())
+      .then(data => setLastSessionDate(data));
+  };
+  useEffect(() => {
+    fetchDayOfLastBoulderSession();
+  }, []);
+
+  useEffect(() => {
+    calculateDateDifference();
+  }, [lastSessionDate]);
+
+  return (
+    <Counter>
+      <p>Last session was </p>
+      <p>
+        <strong>{dateDifference}</strong>
+      </p>
+      <p>ago.</p>
```

Comment:
You are using three `p` because you want to have the content in `strong` in a new line or? I think you could also work with switching the default of display of your `strong` element to `block` and it should work the same. Then you can work with one `p` element instead of three. :)

Reply:
Thank you! I have improved this.